docs/naps/5-new-logo.md

```diff
@@ -0,0 +1,190 @@
+  then apply scale, translation and rotation so that the island is at 45° and
+  the logo fills the space in [[0, 1024], [0, 1024]].
+- NumPy for computation of all the lines and shapes.
+
+Additionally, napari was used extensively during the development, to debug
+errors in calculating the contact points and so on. 😅
+
+It's very fun to work with powers of $\phi$: $1/\phi = \phi - 1$, which implies
+$1/\phi^2 = (\phi - 1)/\phi = 1 - 1/\phi = 2 - \phi$, and so on. Similarly,
+$\phi^2 = 1 + \phi$, which again lets you reduce all polynomials of $\phi$ to
+degree 1.
+
+The original implementation is at https://github.com/jni/new-napari-logo, but
+if this NAP is accepted, it would add the logo-generating script as a gallery
+example.
+
+Another bit of mathematical fun: Veritasium calls $\phi$ "the fiveiest number"
+(in [this fantastic video](https://youtu.be/48sCx-wBs34)): it is the ratio of
+the diagonal of a regular pentagon to its side, and it can be written as
+$\phi = 0.5 + 0.5 \times 5^{0.5}$! So it is a happy not-quite-coincidence that
+this is NAP-5. 😉
+
+## Backward Compatibility
+
+This NAP does not raise backwards compatibility concerns.
+
+## Future Work
+
+As mentioned above, this NAP is about the shapes of the logo, not the colors.
+The colors are sampled from Wikipedia's satellite image of Tabuaeran, except for
+the lagoon, which is the turquoise of the napari-hub and napari sphinx theme.
+It's kinda nice to have those match, but not necessary. We *could* change the
+color to be more natural (again sampled from the satellite) *and* update our
+themes to match. Or we could simply pick a more natural color for the lagoon,
+again from the satellite image, and forget about matching them.
+
+I will take the opportunity to express two strong opinions:
+- I think some form of "ocean purple" is absolutely necessary for the
+  background, as I think it's become part of our identity on account of our
+  existing logo.
+- Whatever exact colors we pick, I think they should continue to evoke the
+  island: ocean, sand, forest, lagoon. There's a lot of range in that space but
+  I think we should keep to that constraint.
+
+If anyone is keen to experiment with different colors I would be glad to pass
+the design baton over.
+
+Other design questions:
+
+- Do we want a slight drop shadow? It is quite standard in macOS icons, and
+  one looks quite sparse without it.
+- Do we want a sunlight reflection/sheen on the ocean part? I'm open to it.
+- For either of those, ideally, we would keep the image processing in Python
+  and napari.
+
+## Alternatives
+
+We could of course go for a more traditionally developed logo. However, I think
+the mathematical story and the ability to generate the logo *using* napari are
+actually important. So I'd love to keep at least a large part of the concept
+for the logo, regardless of where this NAP lands.
+
+It's been suggested that the squircle shape might make it harder to tile a
```

Comment:
I agree that tiling is a nice-to-have, but as you said, the scientific python ecosystem stickers have not really gone with a tile-able standard. Maybe that a hexagonal packing sticker standard would be a good [SPEC](https://scientific-python.org/news/specs-announced-scipy2021/) for the scientific python community ;)